src/index.html

```diff
@@ -1,11 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Document Two Sections</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Excepteur Sint Occaecat</h1>
+    <!-- This is the sidebar area-->
+    <div class="side-bar">
+      <ul>
+        <li><a href="index.html" id="home-link">home</a></li>
```

Comment:
I see you added id values for each. Is there any particular reason to have an id?

You don't have to have an id for every single element in the document. You're doing extra work :)

Usually we only add it if there is a need, like for selecting it for CSS or JavaScript.

Reply:
I use it as a reference just in case in the future, we may want to style each link according to the theme of the website.